vispy/app/backends/tests/test_rfb.py

```diff
@@ -0,0 +1,77 @@
+# This currenly only tests that the backend exists and can be imported ...
+
+import numpy as np
+from vispy import gloo
+from vispy.app import Application, Canvas
+from vispy.app.backends import _jupyter_rfb
+from vispy.testing import requires_application
+
+
+def test_rfb_app():
+
+    # Raw
+    app_backend = _jupyter_rfb.ApplicationBackend()
+
+    # Test that run and quit don't do anything - Jupyter is an interactive session!
+    app_backend._vispy_run()
+    app_backend._vispy_quit()
+
+
+class MyCanvas(Canvas):
+
+    def on_draw(self, event):
+        gloo.set_clear_color((0, 1, 0))
+        gloo.clear()
+
+
+@requires_application()
+def test_rfb_canvas():
+
+    try:
+        import jupyter_rfb
+    except ImportError:
+        return  # only the "all deps" build have jupyter_rfb
```

Comment:
This should be a pytest.skip. You could probably even put it at the module level: https://docs.pytest.org/en/latest/how-to/skipping.html#skipping-test-functions

Reply:
You're right. I think I was in a lazy mood.

Reply:
fixed.